### ISSUE TITLE
Data from The Economist figures can't load?

### ISSUE THREAD

**Issue by k8hertweck** [thebioengineer/tidytuesdayR#20]
This dataset: https://github.com/rfordatascience/tidytuesday/tree/master/data/2019/2019-04-16

**Comment by thebioengineer** [comment]
Could you post your error?

**Comment by k8hertweck** [comment]
Ah, I suppose that would help. I just tried again and they do load, but with the following warning:

```
Warning messages:
1: Missing column names filled in: 'X2' [2], 'X3' [3] 
2: Missing column names filled in: 'X1' [1] 
3: Duplicated column names deduplicated: 'Current-account balance, euro million' => 'Current-account balance, euro million_1' [3], 'Current-account balance, euro million' => 'Current-account balance, euro million_2' [4], 'Current-account balance, euro million' => 'Current-account balance, euro million_3' [5], 'Current-account balance, euro million' => 'Current-account balance, euro million_4' [6], 'Current-account balance, euro million' => 'Current-account balance, euro million_5' [7], 'Current-account balance, euro million' => 'Current-account balance, euro million_6' [8], 'Budget balance, euro million' => 'Budget balance, euro million_1' [10], 'Budget balance, euro million' => 'Budget balance, euro million_2' [11], 'Budget balance, euro million' => 'Budget balance, euro million_3' [12], 'Budget balance, euro million' => 'Budget balance, euro million_4' [13], 'Budget balance, euro million' => 'Budget balance, euro million_5' [14], 'Budget balance, euro million' => 'Budget balance, euro m [... truncated] 
4: Missing column names filled in: 'X1' [1], 'X3' [3], 'X4' [4], 'X5' [5], 'X6' [6] 
```

**Comment by thebioengineer** [comment]
resolved with pull request #28
